test/games/patchers/test_randomprime_patcher.py

```diff
@@ -73,898 +73,18 @@ def test_create_patch_data(test_files_dir, mocker):
-
-                "\n\n&push;&font=C29C51F1;&main-color=#33ffd6;Artifact of Wild&pop;\n"
-                "&push;&main-color=#d4cc33;Prime&pop;'s Tallon Overworld - Transport Tunnel B"
-
-                "\n\n&push;&font=C29C51F1;&main-color=#33ffd6;Artifact of World&pop;\n"
-                "&push;&main-color=#d4cc33;Echoes&pop;'s Ing Hive - Culling Chamber"
-            ),
-            'etankCapacity': 110,
-
-            "mainPlazaDoor": True,
-            "backwardsFrigate": True,
-            "backwardsLabs": True,
-            "backwardsUpperMines": True,
-            "backwardsLowerMines": False,
-            "phazonEliteWithoutDynamo": True,
-
-            'gameBanner': {'description': 'Seed Hash: Putrid Shriekbat Polluted',
-                           'gameName': 'Metroid Prime: Randomizer',
-                           'gameNameFull': 'Metroid Prime: Randomizer - AAAAAAAA'},
-            'heatDamagePerSec': 10.0,
-            'itemMaxCapacity': {'Unknown Item 1': 65536},
-            'mainMenuMessage': f'Randovania v{randovania.VERSION}\nPutrid Shriekbat Polluted',
-            'nonvariaHeatDamage': True,
-            'staggeredSuitDamage': True,
-            'disableItemLoss': True,
-            'startingItems': {
-                'bombs': False,
-                'boostBall': False,
-                'combatVisor': True,
-                'charge': False,
-                'energyTanks': 0,
-                'flamethrower': False,
-                'grapple': False,
-                'gravitySuit': False,
-                'ice': False,
-                'iceSpreader': False,
-                'missiles': 5,
-                'morphBall': True,
-                'phazonSuit': False,
-                'plasma': False,
-                'powerBeam': True,
-                'powerBombs': 0,
-                'scanVisor': True,
-                'spaceJump': True,
-                'spiderBall': False,
-                'superMissile': False,
-                'thermalVisor': False,
-                'variaSuit': False,
-                'wave': False,
-                'wavebuster': False,
-                'xray': False,
-            },
-            'startingMemo': 'Artifact of Sun, 5 Missiles',
-            'startingRoom': 'Tallon Overworld:Landing Site',
-            'warpToStart': False,
-        },
-        "tweaks": {},
-        'levelData': {
-            'Impact Crater': {
-                'rooms': {},
-                'transports': {
-                    'Crater Entry Point': 'Artifact Temple',
-                },
-            },
-            'Phendrana Drifts': phendrana,
-            'Frigate Orpheon': {'transports': {}, 'rooms': {}},
-            'Magmoor Caverns': magmoor,
-            'Phazon Mines': mines,
-            'Tallon Overworld': overworld,
-            'Chozo Ruins': chozo
-        },
-    }
+    # Expected Result
+    file = open(os.path.join("test", "games", "patchers", "randomprime_expected_data.json"), "r")
```

Comment:
`with test_files_dir.joinpath("...").open() as file:`
And move `randomprime_expected_data.json` to be inside the `test_files` folder, though you can keep the `game/patches` path if you want.